docs/pages/deploy-a-cluster/helm-deployments/azure.mdx

```diff
@@ -0,0 +1,325 @@
+pod/teleport-auth-57989d4cb-4q2ds    1/1     Running   0          22h
+pod/teleport-auth-57989d4cb-rtrzn    1/1     Running   0          22h
+pod/teleport-proxy-c6bf55cfc-w96d2   1/1     Running   0          22h
+pod/teleport-proxy-c6bf55cfc-z256w   1/1     Running   0          22h
+
+NAME                        TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)                                                                     AGE
+service/teleport            LoadBalancer   10.40.11.180   34.138.177.11   443:30258/TCP,3023:31802/TCP,3026:32182/TCP,3024:30101/TCP,3036:30302/TCP   22h
+service/teleport-auth       ClusterIP      10.40.8.251    <none>          3025/TCP,3026/TCP                                                           22h
+service/teleport-auth-v13   ClusterIP      None           <none>          <none>                                                                      22h
+service/teleport-auth-v14   ClusterIP      None           <none>          <none>                                                                      22h
+
+NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/teleport-auth    2/2     2            2           22h
+deployment.apps/teleport-proxy   2/2     2            2           22h
+
+NAME                                       DESIRED   CURRENT   READY   AGE
+replicaset.apps/teleport-auth-57989d4cb    2         2         2       22h
+replicaset.apps/teleport-proxy-c6bf55cfc   2         2         2       22h
+```
+
+## Step 4/5. Set up DNS
+
+We'll now set up DNS `A` records for `teleport.example.com` and `*.teleport.example.com`.
+
+```code
+# Change these parameters if you altered them above
+$ NAMESPACE=teleport
+$ RELEASE_NAME=teleport
+$ PREFIX=teleport
+$ ZONE=example.com
+$ ZONE_RG=dns-rg
+$ external_ip="$(kubectl --namespace ${NAMESPACE} get service/${RELEASE_NAME} -o jsonpath='{.status.loadBalancer.ingress[*].ip}')"
+$ az network dns record-set a add-record --resource-group ${ZONE_RG} --zone-name ${ZONE} --record-set-name ${PREFIX} --ipv4-address "${external_ip}"
+$ az network dns record-set a add-record --resource-group ${ZONE_RG} --zone-name ${ZONE} --record-set-name "*.${PREFIX}"" --ipv4-address "${external_ip}"
+```
+
+## Step 5/5. Create a Teleport user
+
+Create a user to be able to log into Teleport. This needs to be done on the Teleport auth server,
+so we can run the command using `kubectl`:
+
+<ScopedBlock scope={["oss"]}>
+```code
+$ kubectl --namespace teleport exec deploy/teleport-auth -- tctl users add test --roles=access,editor
+
+User "test" has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h:
+https://teleport.example.com:443/web/invite/91cfbd08bc89122275006e48b516cc68
+
+NOTE: Make sure teleport.example.com:443 points at a Teleport proxy that users can access.
+```
+</ScopedBlock>
+<ScopedBlock scope={["enterprise", "cloud"]}>
```

Comment:
This is a cluster deployment guide, so doesn't need the Cloud scope

Reply:
In my defense, I copypasted this from one of the other guides LOL

Reply:
Switched to `TabItem`s scoped for oss and enterprise.